### PR TITLE
Presentation: Fix `HiliteSetProvider` stopping from working after `Presentation` is re-initialized

### PR DESCRIPTION
**Pull request by grigasp**

### CODE REVIEW

File: presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts [hunk 1]
```diff
@@ -6,7 +6,6 @@
  * @module UnifiedSelection
  */
 
-import { once } from "lodash";
```
Comment (roluk) [inline presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts:9]: Perhaps we can remove dependency on `lodash` from `package.json`?

**Review by roluk — CHANGES_REQUESTED** [review]

File: presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts (hunk 1 above)
Comment (grigasp) [reply presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts:9]: We probably could, but I don't see that as a high priority thing to do at the moment - it's being used by some of our other dependencies as well.

Comment (grigasp) [reply presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts:9]: BTW, while looking at where we use `lodash` I see `PresentationPropertyDataProvider` also has a similar problem... going to fix it in this PR as well.

Comment (grigasp) [reply presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts:9]: And now I also see that `presentation-frontend` doesn't use `lodash` at all at the moment (was looking at `presentation-components`).. going to make the change as well.

Comment (grigasp) [reply presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts:9]: Fixed the problem in `PresentationPropertyDataProvider` and dropped `lodash` dependency from all presentation packages. Please review.

**Review by pmconne — APPROVED** [review]

File: presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts (hunk 1 above)
Comment (roluk) [reply presentation/frontend/src/presentation-frontend/selection/HiliteSetProvider.ts:9]: Thanks

**Review by roluk — APPROVED** [review]

**Pull request merged**